File: exercicios lp/exercicios lp/ex2.c
#include<stdio.h>

int main(){

        double altura;
        int genero;

        printf("Insira sua altura: ");
        scanf("%lf",&altura);

        printf("Insira seu sexo: ");
        scanf("%d",&genero);

        if(genero == 0){
            int resp;
            resp = (73*altura)-58;
            printf("O seu peso ideal e de %d Kg",resp);

        }
        
        else if(genero ==1){
            int resp;
            resp = (62*altura)-45;
            printf("O seu peso ideal e de %d Kg",resp);

        }

    return 0;
}

File: exercicios lp/exercicios lp/go/ex2.go
package main

import "fmt"

func main() {

	var altura, resp float64

	var genero int

	fmt.Print("Insira sua altura: ")
	fmt.Scan(&altura)

	fmt.Print("Insira seu sexo: ")
	fmt.Scan(&genero)

	if genero == 0 {
		resp = (73 * altura) - 58
		fmt.Printf("O seu peso ideal e de %f Kg", resp)

	} else {
		resp = (62 * altura) - 45
		fmt.Printf("O seu peso ideal e de %f Kg", resp)

	}

}
